client/my-sites/comment/comment-permalink.jsx

```diff
@@ -0,0 +1,46 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get, isUndefined } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import ExternalLink from 'components/external-link';
+import { getSiteComment } from 'state/selectors';
+
+const CommentPermalink = ( { isLoading, permaLink, translate } ) =>
+	! isLoading && (
+		<Card className="comment__comment-permalink">
+			<SectionHeader label={ translate( 'Comment Permalink' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **43** times:
`translate( 'Permalink to this comment' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/comments/add-single-comment-view)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).